paasta_tools/chronos_tools.py

```diff
@@ -505,7 +517,8 @@ def validate(self):
         error_msgs.extend(super(ChronosJobConfig, self).validate())
 
         for param in ['epsilon', 'retries', 'cpus', 'mem', 'disk',
-                      'schedule', 'scheduleTimeZone', 'parents', 'cmd']:
+                      'schedule', 'scheduleTimeZone', 'parents', 'cmd',
```

Comment:
They are in the InstanceConfig. Oh, I just now noticed the "super(ChronosJobConfig, self).validate()" above. This should make this unnecessary. cpus and mem are also already there, so those should also be able to be removed.